src/components/People.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonInfo } from './Loader/PersonInfo';
+import { Loader } from './Loader';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isLoading, setIsLoading] = useState(false);
+  const [noPeopleMessage, setNoPeopleMessage] = useState(false);
```

Comment:
Let's try to remove this state. It should not be necessary to keep this info in state. You can check the array length every time you render the component.